### DISCUSSION TITLE
Love the Representation Additions in 0.14

### DISCUSSION THREAD

**Discussion by vantubbe** [MaartenGr/BERTopic#1074] [General]
@MaartenGr The representation features you added in 0.14 are freaking amazing. MMR alone really helps but the text generated topic labels based on keywords and documents is genius. Love that you can chain them together too. I didn't have much luck with Cohere but OpenAI produces fantastic results. Thank you very much for these additions!

**Comment by MaartenGr** [comment]
Thank you for your kind words! If you ever run into any problems or if you have any suggestions, feel free to reach out.
